doc/api/stream.md

```diff
@@ -494,20 +494,29 @@ added: v11.4.0
 
 Is `true` if it is safe to call [`writable.write()`][stream-write].
 
-##### writable.writableEnded
+##### writable.writableEnding
 <!-- YAML
 added: v12.9.0
 -->
 
 * {boolean}
 
+Is `true` when [`writable.end()`][] has been called and before `ended`.
```

Comment:
It's not clear what `ended` is referring to here.

Reply:
@jasnell update ready